lib/src/r_tree/non_leaf_node.dart

```diff
@@ -119,21 +116,13 @@ class NonLeafNode<E> extends Node<E> {
     return bestNode;
   }
 
-  _convertToLeafNode() {
-    var nonLeafParent = parent as NonLeafNode<E>?;
-    if (nonLeafParent == null) return;
-
-    var newLeafNode = LeafNode<E>(this.branchFactor);
-    newLeafNode.include(this);
-    nonLeafParent.removeChild(this);
-    nonLeafParent.addChild(newLeafNode);
-  }
```

Comment:
For this method, see commentary on 4eca138423e030511d3ab1afd12f114da3a9510f.

Reply:
thanks, I intended to migrate the comments but hadn't yet :D